### DISCUSSION TITLE
Can you make a demo to understand the use of Dialogic 2.0？

### DISCUSSION THREAD

**Discussion by k25961918** [dialogic-godot/dialogic#1932] [Ideas]
Hi~ Thank you very much for your plug-in. For me who can’t write scripts, Dialogic 2.0 is really helpful to me. But I still don't know how to use a lot of features, Have you considered making a demo？The simple use of various features in the demo and plug them into the game will help users better understand Dialogic 2.0。

**Comment by Jowan-Spooner** [comment]
You can check out [this kinda demo project](https://github.com/dialogic-godot/test-project). It's not super up to date, but I will try to push an update when we release alpha 12.

Because dialogic is pretty flexible having one demo only is kinda not enough, so the project has three separate very small demos of different things you can achieve.

**Comment by k25961918** [comment]
Thank you very much
